RELEASES.md

```diff
@@ -50,14 +51,14 @@ Requires JDK 17 or later, tested on JDK 17 and 21.
 Requires JDK 11 or later, tested on JDK 11, 17 and 21.
 Package `nl.nn.adapterframework` is renamed to `org.frankframework`.
 Removed many deprecated features.
+Inside Larva configuration XML files, the `nl.nn.adapterframework.` package should be replaced with `org.frankframework.`. This is due to the package name change in the framework.
```

Comment:
'Should be replaced', en niet meer onderdeel van non backwards compatible changes - betekent dit dat het niet moet?